R/pkgs/report_generation/inst/rmarkdown/chunks/importation_seeding.Rmd

```diff
@@ -0,0 +1,4 @@
+
+We used an air importation model in conjunction with the cumulative confirmed COVID case data from [JHU CSSE COVID-19 Data Portal](https://coronavirus.jhu.edu/map.html) and assumptions in recent travel reductions to estimate the number of COVID case importations into major airports located in `r report_loc` and surrounding areas (`r paste(modeled_states, collapse = ", ")`) from January through March. These importations were used to seed our epidemic model.
```

Comment:
We got a comment from a contractor working with Delaware that they thought the importations only came from China - I don't think the text as is implies that, but maybe worth being more explicit

Reply:
added global